tools/chpldoc/chpldoc.cpp

```diff
@@ -2175,6 +2175,8 @@ int main(int argc, char** argv) {
                          chplEnv->at("CHPL_COMM"),
                          chplEnv->at("CHPL_SYS_MODULES_SUBDIR"),
                          chplModulePath,
+                         std::vector<std::string>(),
+                         std::vector<std::string>(),
```

Comment:
here too, can we use `{}`?

Reply:
Furthermore, perhaps leaving an inline comment with the name of the argument would make it clearer what these vectors are (i.e., `/* somePath */ {}`)